Package.swift

```diff
@@ -37,7 +37,7 @@ let environmentVariables = ProcessInfo.processInfo.environment
 let shouldIncludeDocCPlugin = environmentVariables["INCLUDE_DOCC_PLUGIN"] == "true"
 
 var dependencies: [Package.Dependency] = [
-    .package(url: "https://github.com/quick/nimble", revision: "1f3bde57bde12f5e7b07909848c071e9b73d6edc"),
+    .package(url: "https://github.com/quick/nimble", exact: "13.7.1"),
```

Comment:
I fixed it to commits because tags can be technically moved, but maybe it's too much  😅

Reply:
Yeah, I'd say we should be fine fixing the exact version. It's more explicit and in this case it's for tests only. 
So potentially Nimble moving a tag would only be done to fix something that's broken